internal/converters/converters.go

```diff
@@ -33,7 +33,7 @@ func convertStringArr(ifaceArr []interface{}) []string {
 	return arr
 }
 
-func convertIntArr(ifaceArr []interface{}) []string {
+func ConvertIntArr(ifaceArr []interface{}) []string {
```

Comment:
This would be `IfArrToIntStringArr` (or can it be safely replaced with usage of other converters?)